Package.swift

```diff
@@ -349,13 +351,33 @@ let targets: [Target] = [
             .product(name: "OpenAPIRuntime", package: "swift-openapi-runtime"),
             .product(name: "HTTPTypes", package: "swift-http-types"),
             .product(name: "OpenAPIURLSession", package: "swift-openapi-urlsession"),
-            .product(name: "Rosalind", package: "Rosalind"),
         ],
         exclude: ["OpenAPI/server.yml"],
         swiftSettings: [
             .define("MOCKING", .when(configuration: .debug)),
         ]
     ),
+    .target(
+        name: "TuistCLIServer",
```

Comment:
Nitpick. What do you think if we follow this convention `Tuist{Feature}{Destination}`. For example:
- `TuistServerCore`: Platform-agnostic server-related utilities.
- `TuistServerCLI`: Layer upon `TuistServerCore` extending it with CLI-specific capabilities
- `TuistServerApp`: Layer upon `TuistServerCore` extending it with App-specific capabilities.